music21/__init__.py

```diff
@@ -191,7 +192,80 @@
 from music21.test.testRunner import mainTest  # noqa: E402
 
 # -----------------------------------------------------------------------------
-# this brings all of our own __all__ names into the music21 package namespace
-# pylint: disable=wildcard-import
-from music21 import *  # @UnresolvedImport  # noqa: E402,F403
+# now import all modules so they are accessible from "import music21"
+from music21 import abcFormat  # noqa: E402
+from music21 import alpha  # noqa: E402
+from music21 import analysis  # noqa: E402
+from music21 import audioSearch  # noqa: E402
+from music21 import braille  # noqa: E402
+from music21 import capella  # noqa: E402
+from music21 import chord  # noqa: E402
+from music21 import common  # noqa: E402
+from music21 import converter  # noqa: E402
+from music21 import corpus  # noqa: E402
+from music21 import features  # noqa: E402
+from music21 import figuredBass  # noqa: E402
+from music21 import graph  # noqa: E402
+from music21 import humdrum  # noqa: E402
```

Comment:
This is an improvement over the wildcard, so great. I do think we can get rid of these entirely, though. I have a branch wiping these imports out in order to solve #881, but I haven't rebased it on v.7 yet. 

For instance, I don't see a reason we need to support `humdrum` as a name if a super-user is doing a super-user-python-friend-style `import music21`. If someone wants a clean namespace, they should import `music21.humdrum`, or `music21.humdrum as humdrum` or just the modules they need. I think these imports are the cause of #881, but they are a little tedious to fix because fixing them involves cleaning up other circular imports that are hidden by the deterministic import right here.

Reply:
I'm pretty enamored of the notion that when you import a package you get the package, not a random set of modules that happen to be imported because of dependencies.  I'm not sure that changing music21's behavior to fix a Pyinstaller bug is a tradeoff I'm willing to make.

Reply:
Oh for sure, that would be a terrible outcome. I wasn't proposing that. It looks like I was confused -- I thought that `import music21` was adding `humdrum` to the global namespace. I guess I was wrong.

Your change here was enough to solve the pyinstaller doo-hickey after all! 🎉